docs/resources/view_links.md

```diff
@@ -0,0 +1,113 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "launchdarkly_view_links Resource - launchdarkly"
+subcategory: ""
+description: |-
+  Provides a LaunchDarkly view links resource for managing bulk resource linkage to views.
+  This resource allows you to efficiently link multiple flags (and in the future, segments and metrics) to a specific view. This is particularly useful for administrators organizing resources by team or deployment unit.
```

Comment:
double-checking: do we want to communicate now that views will eventually include segments and metrics?

Reply:
So the current approach is that we won't communicate ahead of time, but we will enable new resources as and when they become available, good callout.